packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -587,42 +587,52 @@ class ServiceExtensionManager {
         break;
       case 'Flutter.ServiceExtensionStateChanged':
         final String name = event.json['extensionData']['extension'].toString();
-        final String valueFromJson =
+        final String valueString =
             event.json['extensionData']['value'].toString();
+        await _updateServiceExtensionForStateChange(name, valueString);
+        break;
+      case 'HttpTimelineLoggingStateChange':
+        final name = extensions.httpEnableTimelineLogging.extension;
+        final valueString = event.json['extensionData']['enabled'].toString();
+        await _updateServiceExtensionForStateChange(name, valueString);
```

Comment:
here and elsewhere. valueString -> value

Reply:
done